src/main/java/pages/Discord.java

```diff
@@ -0,0 +1,72 @@
+package pages;
+
+import helpers.TextTranslation;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+import io.appium.java_client.pagefactory.AppiumFieldDecorator;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import utilities.Constants;
+import utilities.DriverConfiguration;
+import utilities.FrontEndOperation;
+
+import java.time.Duration;
+
+import static org.hamcrest.Matchers.is;
+import static org.openqa.selenium.support.PageFactory.initElements;
+
+public class Discord {
+
+  private WebDriver driver;
+
+  @FindBy(className = "login-button-js")
+  @AndroidFindBy(
+      xpath = "//android.widget.Button[@content-desc=\"Iniciar sesión\"]/android.view.ViewGroup")
+  WebElement iniciarSesion;
+
+  @FindBy(id = "uid_7")
+  @AndroidFindBy(xpath = "//android.widget.EditText[@resource-id=\"login_login_input\"]")
+  WebElement email;
+
+  @FindBy(id = "uid_9")
+  @AndroidFindBy(xpath = "//android.widget.EditText[@resource-id=\"login_password_input\"]")
```

Comment:
find this element by id only, not xpath. Try to avoid xpaths in mobile

Reply:
No funciona con los IDS que lo probé ayer